src/query/test/builder.go

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
nit: 2019?

Reply:
This is a bit of a weird case since it's moving a file (excised the "real" codepath and changed it to strictly a test harness) and then editing it a bit? Not sure what the approach here re: year is, thoughts?